### PR TITLE
seccomp flags rework + minor other build system/repo changes

### PR DESCRIPTION
**Pull request by poettering**
Primarily a fix for #8261.

### CODE REVIEW

**Comment by yuwata** [comment]
lgtm.

**Comment by yuwata** [comment]
I think also TRANSLATORS (recently added by 69a653b64d7c076d41a1b0bd3ce4c7da9ae2999b) should be moved to doc.

File: src/core/dbus-execute.c [hunk 1]
```diff
@@ -1616,7 +1616,7 @@ int bus_exec_context_set_transient_property(
                                 if (r < 0)
                                         return r;
 
-                                r = seccomp_parse_syscall_filter(invert, n, e, c->syscall_filter, c->syscall_whitelist);
+                                r = seccomp_parse_syscall_filter(n, e, c->syscall_filter, (invert ? SECCOMP_PARSE_INVERT : 0) | (c->syscall_whitelist ? SECCOMP_PARSE_WHITELIST : 0));
```
Comment (keszybz) [inline src/core/dbus-execute.c:1619]: Can't say I find this very readable, but OK.

**Comment by poettering** [comment]
Force pushed a new version. Only change: move TRANSLATORS too. PTAL

**Comment by yuwata** [comment]
lgtm.

**Pull request merged**